versiondb/extsort/sort.go

```diff
@@ -0,0 +1,230 @@
+package extsort
```

Comment:
is there a reason to have a custom external sort package instead of importing one?

Reply:
It's simpler and cleaner than alternatives like https://github.com/lanrat/extsort, and no need to worry about third-party maintenence.
At the beginning I also tried to implement [tornament tree](https://en.wikipedia.org/wiki/K-way_merge_algorithm#Tournament_Tree) as recommended by wikipedia, but give up and just stick with heap in stdlib.

Reply:
ok, maybe good to document this in the package main's file

Reply:
done